esmf_regrid/_esmf_sdo.py

```diff
@@ -0,0 +1,225 @@
+# -*- coding: utf-8 -*-
```

Comment:
Is there a specific reason for this line? Is this worth adding to other files?

Reply:
More of a habit. It did have its use in Python 2 and is still useful for my beloved emacs, but I just read up on it and it seems in Python 3 it is no longer expected because utf-8 already is the default. So I'll remove it for consistency.